src/Admin/AbstractAdmin.php

```diff
@@ -1691,7 +1691,7 @@ public function getSubject()
             $id = $this->request->get($this->getIdParameter());
 
             if (null !== $id) {
-                $this->subject = $this->getModelManager()->find($this->class, $id);
+                $this->subject = $this->getObject($id);
```

Comment:
This will trigger extensions, but I think that might be ok.

Reply:
This will serve the same object every time.